examples/common/src/framework.rs

```diff
@@ -29,6 +29,7 @@ pub enum ShaderStage {
 }
 
 pub trait Example: 'static + Sized {
+    const SRGB: bool = true;
```

Comment:
If we make this a function, there doesn't need to be const generics in the example

Reply:
You'd still need const generics as `framework::run` doesn't take arguments.